cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/messaging/aws/sqs/DefaultSqsQueueResourceFactory.java

```diff
@@ -56,7 +56,6 @@ public SqsQueueResource createSqsQueueResource(final String name, final SnsTopic
         final String queueUrl = amazonSqsClient.getQueueUrl(new GetQueueUrlRequest(name)).getQueueUrl();
         logger.info("Using existing SQS queue: " + name);
         final SqsQueueResource sqsQueueResource = new SqsQueueResource(name, queueUrl, amazonSqsClient);
-        subscribeToSnsTopics(name, sqsQueueResource, snsTopics);
```

Comment:
On reflection, the second bullet point above is irrelevant as the method `createSqsQueueResource` is only called from the other factory method `createSqsQueueResourceAndAwsSqsQueueIfAbsent` and the application we are working on currently.